### PR TITLE
Feature/eng-47/Profile Screen

### PR DESCRIPTION
**Pull request by iyiolaosuagwu**
1. developed profile screen
2. developed edit profile screen
3. custom hook for tabs switch 

https://github.com/user-attachments/assets/fa94e470-e79f-46f1-bbb3-3cc4bb1ae6d9

https://github.com/user-attachments/assets/0893ba12-4c69-4758-851d-8c95869231ea

### CODE REVIEW

**Comment by njokuScript** [comment]
Screenshot 2024-12-06 at 11 53 12 AM
That link button changes to this check icon when clicked, it's the URL to your profile.

File: app/(tabs)/profile/_layout.tsx [hunk 1]
```diff
@@ -15,6 +15,14 @@ const ProfileStack = () => {
...
         }}
       />
+
+      <Stack.Screen
+        name="edit-profile"
+        options={{
+          header: (props) => <LQDStackHeader {...props} style={{ paddingTop: 14 }} />,
+          headerShown: false, // fix header issue on this screen
+        }}
+      />
```
Comment (MeisterJustice) [inline app/(tabs)/profile/_layout.tsx]: if `headerShown: false` then there's no need to have `header: (props) => <LQDStackHeader {...props} style={{ paddingTop: 14 }} />`

File: components/image/index.tsx [hunk 2]
```diff
@@ -0,0 +1,70 @@
+import React, { useState } from 'react';
+import FastImage from 'react-native-fast-image';
+import { Pressable, StyleSheet, View } from 'react-native';
+import { LQDImageProps } from './types';
+import { EditProfileIcon } from '@/assets/icons';
+const fallbackImage = 'https://pics.craiyon.com/2023-08-02/7a951cac85bd4aa2b0e70dbaabb8404e.webp';
+const LQDImage = ({
+  style,
```
Comment (MeisterJustice) [inline components/image/index.tsx]: can you add spacings here?

File: screens/profile/index.tsx [hunk 3]
```diff
@@ -1,8 +1,108 @@
...
+    <View style={styles.container}>
+      <View style={styles.topWrapper}>
+        <LQDImage action={() => router.push(`/(tabs)/profile/edit-profile`)} edit height={56} width={56} />
+        <Text style={styles.username}>@jeffing.eth</Text>
+        <View style={{ flexDirection: 'row', gap: 7 }}>
+          <Text style={styles.address}>0xc57...13d4f</Text>
+          <CopyIcon fill="#64748B" />
+        </View>
+        <View style={{ flexDirection: 'row', alignItems: 'center', gap: 9, marginTop: 15, marginBottom: 20 }}>
+          <Pressable style={styles.actionBtn}>
```
Comment (MeisterJustice) [inline screens/profile/index.tsx]: can we remove the inline stylings here?

**Review by MeisterJustice — CHANGES_REQUESTED** [review]

**Review by MeisterJustice — APPROVED** [review]

**Pull request merged**